### PR TITLE
Query multiple table 

### PR DESCRIPTION
**Pull request by HectorTorresE**
This PR modifies these files:

- schema.sql Add tables species and owners, modifies animals table to reference foreign keys
- data.sql Add data to tables owners and species, update animals' data to reference external tables
- queries.sql Query multiple tables using

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Boarmon', '2005-06-07', 7,true, 20.4);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Blossom', '1998-10-13', 3,true, 17);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Ditto', '2022-05-14', 4,true, 22);
\ No newline at end of file
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith 34', 34),
```
Comment (Gambit142) [inline data.sql]: - [x] You have a typographical error when populating data into the `owners` table for Sam Smith. It should be `Sam Smith` and not `Sam Smith 34`. This affects the results gotten from some queries. **_For example, the query that asks for all owners and their animals, including those that don't own any animal_**. Kindly correct it 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @HectorTorresE 👋, 

Good job so far!

## Highlights
- Created `species` and `animals` table ✔️ 
- Updated foreign keys in the necessary tables ✔️ 
- Inserted data into the `Owners`, `Species` and Animals tables ✔️  
- Wrote the queries according to the project's requirements ✔️ 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- I noticed the screenshots you attached to the pull request summary are different from the results of the queries. Kindly update them. 👍 

image

image


- _Kindly check the comments under the review for all other required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @HectorTorresE 🙋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] please work on the change requested by the previous code reviewer, it's important to use accurate values in your DB because corrupted data can lead to data loss forever 🙂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello @HectorTorresE 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**